tests/fixtures/account/update-bad-email.xml

```diff
@@ -7,12 +7,12 @@ Content-Type: application/xml; charset=utf-8
 
 <?xml version="1.0" encoding="UTF-8"?>
 <account>
-  <username>shmohawk58</username>
+  <accept_language>en-US</accept_language>
+  <company_name>Home Box Office</company_name>
   <email>larry.david</email>
-  <first_name>Lärry</first_name>
```

Comment:
what is the reason for all these fixture changes? fixing failing specs?

Reply:
as I did this change to have the attributes of the xml more stable, I have to reorder the xml. because the problem was, sometimes the body of the request was generated like this.

```xml
<ramp_intervals>
  <ramp_interval>
    <unit_amount_in_cents></unit_amount_in_cents>
    <billing_cycle></billing_cycle>
  </ramp_interval>
  ...
</ramp_intervals>
```

and sometimes.
```xml
<ramp_intervals>
  <ramp_interval>
    <billing_cycle></billing_cycle>
    <unit_amount_in_cents></unit_amount_in_cents>
  </ramp_interval>
  ...
</ramp_intervals>
```
So I have to sorted the `attributes` to have the xml body attributes sorted.

Reply:
gotcha